Failed to import the site module
Traceback (most recent call last):
  File "/home/yuntian/anaconda3/envs/main/lib/python3.6/site.py", line 75, in <module>
    import _sitebuiltins
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 764, in get_code
  File "<frozen importlib._bootstrap_external>", line 833, in get_data
KeyboardInterrupt
